projects/1.18/assets/moa-decor-science/moa_decor_science/lang/zh_cn.json

```
@@ -0,0 +1,154 @@
+  "block.moa_decor_science.globoterraqueo": "桌上地球仪",
+  "block.moa_decor_science.jeringa": "注射器",
+  "block.moa_decor_science.jeringaamarilla": "装有黄色物质的注射器",
+  "block.moa_decor_science.jeringaazul": "装有蓝色物质的注射器",
+  "block.moa_decor_science.jeringaazulclara": "装着淡蓝色物质的注射器",
+  "block.moa_decor_science.jeringablanca": "装有白色物质的注射器",
+  "block.moa_decor_science.jeringacafe": "装有棕色物质的注射器",
+  "block.moa_decor_science.jeringacian": "装有青色物质的注射器",
+  "block.moa_decor_science.jeringagris": "装有灰色物质的注射器",
+  "block.moa_decor_science.jeringagrisclara": "装有淡灰色物质的注射器",
+  "block.moa_decor_science.jeringamagenta": "装有品红色物质的注射器",
+  "block.moa_decor_science.jeringamorada": "装有紫色物质的注射器",
+  "block.moa_decor_science.jeringanaranja": "装有橙色物质的注射器",
+  "block.moa_decor_science.jeringanegra": "装有黑色物质的注射器",
+  "block.moa_decor_science.jeringaroja": "装有红色物质的注射器",
+  "block.moa_decor_science.jeringarosa": "装有粉色物质的注射器",
+  "block.moa_decor_science.jeringaverde": "装有绿色物质的注射器",
+  "block.moa_decor_science.jeringaverdelima": "装有黄绿色物质的注射器",
+  "block.moa_decor_science.lectordesignosvitales": "生命体征监测仪",
+  "block.moa_decor_science.libro": "书",
+  "block.moa_decor_science.libroabierto": "打开的书",
+  "block.moa_decor_science.libros": "一摞书",
+  "block.moa_decor_science.librosapilados": "一排书",
+  "block.moa_decor_science.libroytintero": "书和墨水瓶",
+  "block.moa_decor_science.lupa": "放大镜",
+  "block.moa_decor_science.mapadeltesoro": "藏宝图",
+  "block.moa_decor_science.matrazb": "烧瓶",
+  "block.moa_decor_science.matrazbamarillo": "装有黄色物质的烧瓶",
+  "block.moa_decor_science.matrazbazul": "装有蓝色物质的烧瓶",
+  "block.moa_decor_science.matrazbazulclaro": "装有淡蓝色物质的烧瓶",
+  "block.moa_decor_science.matrazbblanco": "装有白色物质的烧瓶",
+  "block.moa_decor_science.matrazbcafe": "装有棕色物质的烧瓶",
+  "block.moa_decor_science.matrazbcian": "装有青色物质的烧瓶",
+  "block.moa_decor_science.matrazbgris": "装有灰色物质的烧瓶",
+  "block.moa_decor_science.matrazbgrisclaro": "装有淡灰色物质的烧瓶",
+  "block.moa_decor_science.matrazbmagenta": "装有品红色物质的烧瓶",
+  "block.moa_decor_science.matrazbmorado": "装有紫色物质的烧瓶",
+  "block.moa_decor_science.matrazbnaranja": "装有橙色物质的烧瓶",
+  "block.moa_decor_science.matrazbnegro": "装有黑色物质的烧瓶",
+  "block.moa_decor_science.matrazbrojo": "装有红色物质的烧瓶",
+  "block.moa_decor_science.matrazbrosa": "装有粉色物质的烧瓶",
+  "block.moa_decor_science.matrazbverde": "装有绿色物质的烧瓶",
+  "block.moa_decor_science.matrazbverdelima": "装有黄绿色物质的烧瓶",
+  "block.moa_decor_science.microscopio": "显微镜",
+  "block.moa_decor_science.pergamino": "羊皮纸",
+  "block.moa_decor_science.pergaminoenrollado": "羊皮卷",
+  "block.moa_decor_science.pergaminos": "一堆羊皮卷",
+  "block.moa_decor_science.pizarra": "黑板",
+  "block.moa_decor_science.pizarraabecedario": "写有英文字母的黑板",
+  "block.moa_decor_science.pizarraadn": "画有DNA图解的黑板",
+  "block.moa_decor_science.pizarramatematicas": "写有数学等式的黑板",
+  "block.moa_decor_science.pizarravolumenes": "写有体积公式的黑板",
+  "block.moa_decor_science.portapapeles": "写字板",
+  "block.moa_decor_science.sueroamarillo": "挂着黄色血清的输液架",
```

```suggestion
  "block.moa_decor_science.sueroamarillo": "挂有黄色物质的输液架",
```
原文`Pedestal con suero`+颜色
Pedestal=底座，suero=血清，Pedestal con suero=带有血清的底座=输液架
于是suero的引申义就是输液，类似还有porta suero=便携输液架